internal/util/template.go

```diff
@@ -106,6 +106,11 @@ func valueIfTemplateFunc(value interface{}) (string, error) {
 			return "", nil
 		}
 
+		m, ok := reflect.TypeOf(value).MethodByName("GetValue")
+		if ok {
+			value = m.Func.Call([]reflect.Value{reflect.ValueOf(value)})[0].Interface()
```

Comment:
Could you pls explain to me what's happening here and why exactly? I'd like to learn :)

Reply:
If a structure has `GetValue` method - call this method to get a value. This is needed because after moving to golang/protobuf there are wrappers for types:
https://github.com/golang/protobuf/blob/master/ptypes/wrappers/wrappers.pb.go

The same is for methods in `.tpl` files.

Reply:
Thanks!